### PR TITLE
Feature: Display release notes after updating app

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Fixed references to update service

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

**Comment by d2dyno1** [comment]
Nice!

File: src/Files.App/ViewModels/ToolbarViewModel.cs [hunk 1]
```diff
@@ -379,6 +403,17 @@ public ToolbarViewModel()
 
 			SearchBox.Escaped += SearchRegion_Escaped;
 			UserSettingsService.OnSettingChangedEvent += UserSettingsService_OnSettingChangedEvent;
+
+			PullReleaseNotes();
+		}
+
+		private void PullReleaseNotes()
```
Comment (d2dyno1) [inline src/Files.App/ViewModels/ToolbarViewModel.cs:410]: This will most likely become async

File: src/Files.App/ViewModels/ToolbarViewModel.cs [hunk 2]
```diff
@@ -379,6 +403,17 @@ public ToolbarViewModel()
 
 			SearchBox.Escaped += SearchRegion_Escaped;
 			UserSettingsService.OnSettingChangedEvent += UserSettingsService_OnSettingChangedEvent;
+
+			PullReleaseNotes();
```
Comment (d2dyno1) [inline src/Files.App/ViewModels/ToolbarViewModel.cs:407]: Don't call it from the constructor!!

File: src/Files.App/ViewModels/ToolbarViewModel.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/ViewModels/ToolbarViewModel.cs]: I created a new service for this

File: src/Files.App/ViewModels/ToolbarViewModel.cs (hunk 2 above)
Comment (yaira2) [reply src/Files.App/ViewModels/ToolbarViewModel.cs]: Removed

File: src/Files.App/App.xaml.cs [hunk 3]
```diff
@@ -209,6 +210,10 @@ private static async Task InitializeAppComponentsAsync()
 			await updateService.CheckForUpdates();
 			await updateService.DownloadMandatoryUpdates();
 
+			// Download release notes
+			var releaseNotesService = Ioc.Default.GetRequiredService<IReleaseNotesService>();
```
Comment (yaira2) [inline src/Files.App/App.xaml.cs:214]: Should this be moved to line 214?

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**